bundles/org.openhab.core/src/test/java/org/eclipse/smarthome/core/library/types/DateTimeTypeTest.java

```diff
@@ -38,6 +39,7 @@
  * @author Thomas.Eichstaedt-Engelen - Initial contribution
  * @author Gaël L'hopital - Added Timezone and Milliseconds
  * @author Erdoan Hadzhiyusein - Added ZonedDateTime tests
+ * @author Laurent Garnier - Enhanced tests
  */
 @RunWith(Parameterized.class)
 public class DateTimeTypeTest {
```

Comment:
Looking at the properties of the inner class `ParameterSet` does it make sense to add nullness annotations here?

Reply:
Done